Here's what I get
```
======================================================================
ERROR: test_hierarchy.test_linkage_from_reachability
Y encodes weighted distances for pairs:
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/gnu/store/dx60jjrjvh44q88r6kc2m1jscfda5r44-profile/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/srv/home/uqbwoodc/git/GroopM/groopm/tests/test_hierarchy.py", line 386, in test_linkage_from_reachability
    assert_true(equal_arrays(linkage_from_reachability(d), Z),
TypeError: linkage_from_reachability() takes exactly 2 arguments (1 given)

----------------------------------------------------------------------
Ran 28 tests in 268.352s

FAILED (errors=1)
```
Looks like a legitimate error. For the rest, maybe some annotation so that the tests are skipped by default is in order.

--- a/groopm/tests/test_hierarchy.py
+++ b/groopm/tests/test_hierarchy.py
@@ -383,7 +383,7 @@ def test_linkage_from_reachability():
                   [2., 7., 70.0, 5.]])
     
     (o, d) = distance.reachability_order(Y)
-    assert_true(equal_arrays(linkage_from_reachability(d), Z),
+    assert_true(equal_arrays(linkage_from_reachability(o, d), Z),
                 "returns linkage corresponding to reachability ordering")
                         
     
@@ -434,7 +434,7 @@ def test_linkage_from_reachability():
                   [10., 11., 3., 7.]])
                   
     (o, d) = distance.reachability_order(Y)
-    assert_true(equal_arrays(linkage_from_reachability(d)[:, 2],
+    assert_true(equal_arrays(linkage_from_reachability(o, d)[:, 2],
                              Z[:, 2]),
                 "returns linkage with correct heights for a moderately complex "
                 "hierarchy")

